src/category_theory/limits/limits/over.lean

```diff
@@ -0,0 +1,154 @@
+-- Copyright (c) 2018 Scott Morrison. All rights reserved.
```

Comment:
Huh what happended here? I guess you need to remove this file (it looks like a copy of ` src/category_theory/limits/over.lean`, note the two `limits` directories in this patch`)

It's good to have the check for double names in `leanpkg test` :-)